__tests__/app.test.js

```diff
@@ -78,3 +78,41 @@ describe("GET /api", () => {
       });
   });
 });
+describe("GET /api/articles/:articleId", () => {
+  test("should return 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should return an object with all expected properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article[0];
```

Comment:
Suggestion: to improve this test you could test using the actual data of the article object since you know it's ID